sarsen/apps.py

```diff
@@ -6,6 +7,21 @@
 
 from . import geocoding, orbit, scene
 
+logger = logging.getLogger(__name__)
+
+
+GridParamsType = T.TypedDict(
```

Comment:
Mm. I'm not at all a fan of type definitions that are used only once.

I would prefer to have normal keywords and return the ugly dictionary.